entrypoints/background.ts

```diff
@@ -25,22 +25,19 @@ export default defineBackground(() => {
     throw new Error('Unknown response format');
   }
 
-  browser.runtime.onMessage.addListener(async (message: ExtensionMessage, sender, sendResponse) => {
+  browser.runtime.onMessage.addListener((message: ExtensionMessage, sender, sendResponse) => {
     switch (message.action) {
       case ExtensionMessageAction.Translate:
-        try {
-          const { text, source_lang, target_lang } = message.payload;
-          const translation = await getTranslation(text, source_lang, target_lang);
-          return {
-            error: null,
-            response: translation
-          };
-        } catch (e) {
-          return {
-            error: e,
-            response: null
-          };
-        }
+        (async () => {
```

Comment:
I had to do this because the local server was not transferring data from the background to the content script.

Reply:
> I had to do this because the local server was not transferring data from the background to the content script.

Looks like this might be related to a bug in Chrome: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessage#sending_an_asynchronous_response_using_a_promise

Learned something new today!